### DISCUSSION TITLE
"forge -m" not working (begginer question)

### DISCUSSION THREAD

**Discussion by AugustoDEV1** [foundry-rs/foundry#5280] [Help] [answered]
When I run "forge -m" to execute a specific function it says error: unexpected argument '-m' found. I believe this isn't able on my device, can anyone provide a solution? many thankss

**Comment by alfheimrShiven** [comment]
I'm facing the same issue as well. I even tried running the following:
```
forge --match-contract {contractName} --match-test {testName}
```
but no luck.
I'm on `forge 0.2.0`

Thanks!

**Reply by mds1** [reply]
You are missing the `test` part of the command

**Reply by alfheimrShiven** [reply]
My bad. Thanks it works @mds1

**Comment by mds1** [comment, marked as answer]
@AugustoDEV1 see https://github.com/foundry-rs/foundry/discussions/5268#discussioncomment-6336903 for the new flag 

Is there documentation somewhere that says `-m` which needs to be updated?

**Reply by AugustoDEV1** [reply]
Well done! thanks @mds1
